src/components/Checkbox/index.test.tsx

```diff
@@ -26,9 +27,9 @@ describe('Checkbox component', () => {
     );
     const input = screen.getByLabelText('change callback', { exact: false });
     expect(changeFn).not.toHaveBeenCalled();
-    fireEvent.click(input);
+    userEvent.click(input);
     expect(input).toBeChecked();
-    fireEvent.change(input, { target: input });
+    userEvent.type(input, 'change callback');
```

Comment:
You see why this change is better right? It's a lot more coherent.

Reply:
yeah, the syntax here is definitely more readable